src/app/(auth)/sign-in/components/EmailCheckModalContainer.tsx

```diff
@@ -0,0 +1,60 @@
+import Image from "next/image";
+import Button from "@/components/Button";
+
+function EmailCheckModalContainer() {
```

Comment:
None of these are actually modals so I don't think it's appropriate to name it as such. I think you can get rid of modal everywhere you have it in the name and it'll work out just like that. For example, name for this would be EmailCheckContainer.